app/components/MovieCard/MovieCard.tsx

```diff
@@ -17,7 +17,7 @@ const MovieCard = ({
   MenuPopoverLinks,
   extraClasses,
 }: MovieCardtypes) => {
-  const MovieCardClass = classNames("px-2 relative w-[160px] h-[320px]");
+  const MovieCardClass = classNames("px-2 relative w-[160px] h-[380px]");
   return (
     <div className={`${MovieCardClass} ${extraClasses ?? ""} `}>
```

Comment:
This card should be wrapped into a Link so that if user click on it we can navigate them to the sub page?